debian/configs/nuc7i7dnh/shared.xml

```diff
@@ -0,0 +1,310 @@
+<acrn-config>
```

Comment:
@NanlinXie @junjiemao1 I'm going to guess these scenario XML files will need conversion to the v3.0 format?

Reply:
These scenario configs are in fact copied from configs of misc/config_tools/data and only slightly modified:
```
user@host:~/acrn-work/test/acrn-hypervisor$ diff debian/configs/nuc7i7dnh/shared.xml misc/config_tools/data/nuc11tnbi5/shared.xml
81c81,82
<       <bootargs>console=tty0 console=ttyS0,115200n8 hvlog=2M@0x100000 memmap=2M$0x100000</bootargs>
---
>       <bootargs>rw rootwait root=/dev/nvme0n1p2 console=tty0 console=ttyS0 consoleblank=0 no_timer_check quiet loglevel=3
>         i915.nuclear_pageflip=1 swiotlb=131072</bootargs>
106c107
<       <pci_dev>00:02.0 VGA compatible controller: Intel Corporation UHD Graphics 620 (rev 07)</pci_dev>
---
>       <pci_dev>00:02.0 VGA compatible controller: Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics] (rev 01)</pci_dev>
```
Are these source configs I used not ACRN 3.0 compliant?